package.json

```diff
@@ -55,9 +55,11 @@
 		"eslint-import-resolver-node": "^0.3.4",
 		"eslint-import-resolver-typescript": "^2.3.0",
 		"eslint-plugin-import": "^2.22.1",
+		"eslint-plugin-jsx-a11y": "^6.4.1",
```

Comment:
the react plugins shouldn't be needed, i removed them because the latest airbnb config doesn't require them anymore